context.go

```diff
@@ -15,12 +16,41 @@ import (
 	"zgo.at/zhttp/ctxkey"
 )
 
-// Version of GoatCounter; set at compile-time with:
-//
-//   -ldflags="-X zgo.at/goatcounter/v2.Version=…"
-
+// Version of GoatCounter.
```

Comment:
I want to retain this comment, because you still need to set it for tagged releases (e.g. `2.5.0`).

Reply:
Done